sqldelight-idea-plugin/src/test/kotlin/app/cash/sqldelight/intellij/autocomplete/AutocompleteTests.kt

```diff
@@ -113,9 +113,10 @@ class AutocompleteTests : SqlDelightFixtureTestCase() {
     )
 
     myFixture.complete(BASIC, 1).let {
-      assertThat(it).hasLength(2)
+      assertThat(it).hasLength(3)
       assertThat(it[0].lookupString).isEqualTo("test")
       assertThat(it[1].lookupString).isEqualTo("test2")
+      assertThat(it[2].lookupString).isEqualTo("sqlite_sequence")
```

Comment:
I don't think I want these autocompleting 🤔 we do this for synthetic columns like "rowid" and in general I think I want to take a stance that we only want user driven data to be powering the IDE.

If you type it in manually it should work, and columns could autocomplete as normal if you've selected from the table

Reply:
Okay 🤷🏻‍♂️ Just to mention it: IntelliJ Ultimate adds autocompleting for system elements.
<img width="1920" alt="Bildschirmfoto 2023-06-27 um 13 44 20" src="https://github.com/cashapp/sqldelight/assets/22521688/6ce0c923-ce21-41c3-96e5-dc895adc06d9">

Reply:
the ultimate strategy would to make this a setting in the IDE if you want it or no and default to off

Reply:
(and i find that autocomplete personally extremely overwhelming)

Reply:
We put the system tables into the schema, so we would need to add a hidden flag at sql-psi I guess.

Reply:
Add a settings panel, which defaults hide system tables.